From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 9 May 2024 17:47:04 +0200
Subject: girepository/introspection: Properly check for sanitizer value

The sanitizer option is set to 'none' when not used

Origin: upstream, 2.80.3, commit:5fc08f60b57e01b11c2a4d707121d8cd38979f07
---
 girepository/introspection/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/girepository/introspection/meson.build b/girepository/introspection/meson.build
index 9405686..b23d048 100644
--- a/girepository/introspection/meson.build
+++ b/girepository/introspection/meson.build
@@ -9,7 +9,7 @@ gi_gen_shared_sources = [
 
 gi_gen_env_variables = environment()
 
-if get_option('b_sanitize') != ''
+if get_option('b_sanitize') != 'none'
   gi_gen_env_variables.append(
     'ASAN_OPTIONS', 'verify_asan_link_order=0', separator: ',')
 endif
